src/test/resources/saReport.txt

```diff
@@ -0,0 +1,31 @@
++ Test security analysis report
+  + AC security analysis on network 'sim1'
+    + Network CC0 SC0
+      + Post loading processing
+         Network has 4 buses and 4 branches
+         Network balance: active generation=607.0 MW, active load=600.0 MW, reactive generation=0.0 MVar, reactive load=200.0 MVar
+      + Pre-contingency simulation
+         AC load flow complete with NR status 'CONVERGED'
```

Comment:
@annetill @flo-dup here we can see an issue with reporter API, we don't keep sequence of reports and sub reporters